digits/model/images/classification/test_views.py

```diff
@@ -317,6 +317,20 @@ def check_select_gpus(self, gpu_list):
         job_id = self.create_model(select_gpus_list=','.join(gpu_list), batch_size=len(gpu_list))
         assert self.model_wait_completion(job_id) == 'Done', 'create failed'
 
+    def test_mean_image(self):
+        options = {
+            'use_mean': 'image',
```

Comment:
if my understanding is correct the default value for `use_mean` is `'image'`. Therefore, is this test not already covered by the default case?
Either way it is worth considering adding a test case for `use_mean='none'`